supportedBackends/aws/src/main/scala/cromwell/backend/impl/aws/AwsBatchJob.scala

```diff
@@ -122,7 +122,7 @@ final case class AwsBatchJob(jobDescriptor: BackendJobDescriptor, // WDL/CWL
     |"
     |cat $dockerStderr
     |echo "--$boundary--"
-    |exit $$(cat $dockerRc)
+    |exit 0
```

Comment:
If something went wrong it would have already exited with a non-zero exit code. Previously it was reporting the exit code of the underlying command, which is already correctly captured in the rc file

Reply:
Ah I see. LGTM.